en/docs/testing/test-websocket-endpoints-via-the-websocket-console.md

```diff
@@ -0,0 +1,37 @@
+
+# Test WebSocket Endpoints via the WebSocket Test Console
+
+Choreo provides an integrated WebSocket Console to test publicly exposed WebSocket endpoints of Service components you create and deploy. The WebSocket test Console allows you to connect to the service and interactively pass messages. As Choreo uses OAuth2.0 authentication by default to secure WebSocket APIs, the GraphQL Console generates test keys to test APIs.
+
+Follow the steps below to test a WebSocket endpoint via the WebSocket Console:
+
+1. Sign in to the [Choreo Console](https://console.choreo.dev/).
+
+2. In the **Component Listing** pane, click on the component you want to test.
+
+3. Click **Test** in the left navigation menu and then click **Console**. This opens the **GraphQL Console** pane.
```

Comment:
We need to replace all occurrences of the term "GraphQL" with "WebSocket" right?